Dockerfile

```diff
@@ -83,6 +83,8 @@ RUN chgrp -R 0 /app /venv /web && \
 USER 1001
 
 ENV HEARTBEAT_SEVERITY major
+ENV HOUSEKEEPING_EXPIRED_HOURS 2
+ENV HOUSEKEEPING_INFO_HOURS 12
```

Comment:
Can we call them `HK_EXPIRED_DELETE_HRS` and `HK_INFO_DELETE_HRS`?

Reply:
Done!